django/db/backends/sqlite3/features.py

```diff
@@ -9,6 +9,14 @@
 from .base import Database
 
 
+def supports_cloning():
+    if (int(platform.python_version().startswith('2') or
+            int(platform.python_version_tuple()[1]) < 7)) \
+            and platform.system() == 'Windows':
+        return False
+    return True
```

Comment:
OK, we're talking about https://bugs.python.org/issue27645, so for this we should check both `SQLite` and Python versions:
```suggestion
    return PY37 and Database.sqlite_version_info >= (3, 6, 11)
```

Reply:
Wouldn't this disable SQLite cloning on Linux with 3.6?